low-level/framework/base/sspl_constants.py

```diff
@@ -37,6 +37,7 @@
 CLI_RESOURCE_PATH = "/opt/seagate/%s/sspl/low-level/tests/manual" % (PRODUCT_FAMILY)
 DATA_PATH = "/var/%s/sspl/data/" % (PRODUCT_FAMILY)
 IEM_DATA_PATH = "/var/%s/sspl/data/iem/sspl_iems"  %(PRODUCT_FAMILY)
+IEM_INIT_FILE = "/var/%s/sspl/data/iem/iem_init" %(PRODUCT_FAMILY)
```

Comment:
This marker file is know init status, but its created only on failure, so why not name it accordingly, init_failure. Same applies to constant name

Reply:
renamed the file as suggested